source/navitiacommon/navitiacommon/models.py

```diff
@@ -487,3 +487,27 @@ class BillingPlan(db.Model):
     @classmethod
     def get_default(cls, end_point):
         return cls.query.filter_by(default=True, end_point=end_point).first()
+
+
+class Autocomplete(db.Model, TimestampMixin):
+    id = db.Column(db.Integer, primary_key=True)
+    shape = db.Column(db.Text, nullable=False, unique=True)
+    street = db.Column(db.Enum('BANO', 'OSM', name='source_street'), nullable=True)
+    address = db.Column(db.Enum('BANO', 'OpenAddresses', name='source_address'), nullable=True)
```

Comment:
We will at least need OSM too.
do we want the ```nullable=True```? The only use will be to disable some type of data